graphbolt/include/graphbolt/sampled_subgraph.h

```diff
@@ -113,6 +113,24 @@ struct SampledSubgraph : torch::CustomClassHolder {
    * subgraph.
    */
   torch::optional<torch::Tensor> type_per_edge;
+
+  /**
+   * @brief Version number to indicate graph version in serialization and
+   * deserialization.
+   */
+  static constexpr int64_t kSampledSubgraphSerializeVersionNumber = 1;
```

Comment:
Does external class need to access this field?

Reply:
No I think. Should it be set to private and non-static?

Reply:
move it to sampled_subgraph.cc within the namespace should be fine.